tests/UI/campaigns/functional/FO/03_userAccount/01_orderHistory/01_orderDetails/01_sendMessage.js

```diff
@@ -0,0 +1,249 @@
+  });
+
+  describe('Create order in FO', async () => {
+    it('should go to FO page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToFoToOrder', baseContext);
+
+      await foHomePage.goToFo(page);
+      await foHomePage.changeLanguage(page, 'en');
+
+      const isHomePage = await foHomePage.isHomePage(page);
+      await expect(isHomePage, 'Fail to open FO home page').to.be.true;
+    });
+
+    it('should go to login page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToLoginPageFoToOrder', baseContext);
+
+      await foHomePage.goToLoginPage(page);
+      const pageTitle = await foLoginPage.getPageTitle(page);
+      await expect(pageTitle, 'Fail to open FO login page').to.contains(foLoginPage.pageTitle);
+    });
+
+    it('should sign in with default customer', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'sighInFoToOrder', baseContext);
+
+      await foLoginPage.customerLogin(page, DefaultCustomer);
+      const isCustomerConnected = await foLoginPage.isCustomerConnected(page);
+      await expect(isCustomerConnected, 'Customer is not connected').to.be.true;
+    });
+
+    it('should create an order', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'createOrder', baseContext);
+
+      // Go to home page
+      await foLoginPage.goToHomePage(page);
+
+      // Go to the first product page
+      await foHomePage.goToProductPage(page, 1);
+
+      // Add the created product to the cart
+      await productPage.addProductToTheCart(page);
+
+      // Proceed to checkout the shopping cart
+      await cartPage.clickOnProceedToCheckout(page);
+
+      // Address step - Go to delivery step
+      const isStepAddressComplete = await checkoutPage.goToDeliveryStep(page);
+      await expect(isStepAddressComplete, 'Step Address is not complete').to.be.true;
+
+      // Delivery step - Go to payment step
+      const isStepDeliveryComplete = await checkoutPage.goToPaymentStep(page);
+      await expect(isStepDeliveryComplete, 'Step Address is not complete').to.be.true;
+
+      // Payment step - Choose payment step
+      await checkoutPage.choosePaymentAndOrder(page, PaymentMethods.wirePayment.moduleName);
+      const cardTitle = await orderConfirmationPage.getOrderConfirmationCardTitle(page);
+
+      // Check the confirmation message
+      await expect(cardTitle).to.contains(orderConfirmationPage.orderConfirmationCardTitle);
+    });
+
+    it('should sign out from FO', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'sighOutFO', baseContext);
+
+      await orderConfirmationPage.logout(page);
+      const isCustomerConnected = await orderConfirmationPage.isCustomerConnected(page);
+      await expect(isCustomerConnected, 'Customer is connected').to.be.false;
+    });
+  });
+  describe('Check invoice file in BO', async () => {
```

Comment:
```suggestion
  });
  
  describe('Check invoice file in BO', async () => {
```

Add empty line for readability